storage/integration_test.go

```diff
@@ -2742,24 +2742,34 @@ func TestIntegration_RequesterPays(t *testing.T) {
 	// - (1b) must NOT have that permission on (3a).
 	// - (1a) must NOT have that permission on (3b).
 
-	const wantErrorCode = 400
-
 	ctx := context.Background()
+	h := testHelper{t}
 
-	// Start first client
-	client := testConfig(ctx, t)
-	defer client.Close()
+	// Start client with mainUserEmail creds
+	mainUserClient := testConfig(ctx, t)
+	defer mainUserClient.Close()
 
-	h := testHelper{t}
+	// Get user emails from credentials
+	jwt, err := testutil.JWTConfig()
+	if err != nil {
+		t.Fatal(err)
```

Comment:
Add some text to the errors here and below to make it clear where the failure occurred.